keyboards/bastardkb/dilemma/3x5_3/config.h

```diff
@@ -42,3 +42,6 @@
 #define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
 #define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_LED GP17
 #define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 500U
+
+// Reduce soft serial speed: Work around rp2040 issues
+#define SELECT_SOFT_SERIAL_SPEED 4
```

Comment:
As other, I suspect that this is fixed.  But just in case...
```suggestion
#define SERIAL_USART_SPEED 38400
```